mysite/settings.py

```diff
@@ -110,7 +110,7 @@
 LOGIN_REDIRECT_URL = '/'
 
 
-DATABASES['default'] = dj_database_url.config()
+DATABASES = {'default': dj_database_url.config()}
```

Comment:
[Move this to DATABASES declarations above. (L79)](#diff-7de95d087d95aaffa2c99d4efcfa9e8eR79)